src/tests/test_rest_framework_dso/test_serializers.py

```diff
@@ -72,3 +73,45 @@ def test_pagination_many(api_rf, movie):
             "category": [{"name": "bar"}],
         },
     }
+
+
+@pytest.mark.django_db
+def test_fields_limit_works(api_rf, movie):
+    """Prove that serializer can limit output fields."""
+    django_request = api_rf.get("/", {"fields": "name"})
+    request = Request(django_request)
+    queryset = Movie.objects.all()
+
+    serializer = MovieSerializer(many=True, instance=queryset)
+    serializer.context["request"] = request
```

Comment:
Small catch: couldn't you call `MovieSerializer(many=True, instance=queryset, context={"request": request})`?